data/src/main/java/com/anytypeio/anytype/data/auth/repo/block/BlockDataRepository.kt

```diff
@@ -782,7 +782,7 @@ class BlockDataRepository(
 
     override suspend fun addObjectToSpace(
         command: Command.AddObjectToSpace
-    ): Pair<Id, ObjectWrapper.Type> = remote.addObjectToSpace(command)
+    ): Pair<Id, Struct?> = remote.addObjectToSpace(command)
```

Comment:
do we really need nullable Struct here? let's make not-nullable

Reply:
Yes, we do.  According to the protocol, workspaceObjectAdd.Response can return details as `Map<String, *>?`